apps/sms/index.html

```diff
@@ -128,6 +129,7 @@
   </head>
 
   <body role="application" class="skin-comms theme-communications">
+    <iframe src="/views/shared/shim_host.html" class="shim-host" height="0" width="0"></iframe>
```

Comment:
I'm thinking the same thing about hiding the iframe, will give it a try for all the scenario by raptor.